src/sbvr-api/permissions.ts

```diff
@@ -1256,6 +1256,16 @@ const $getApiKeyPermissions = (() => {
 											},
 										},
 									},
+									$or: [
+										{
+											khr: { expiry_date: null },
```

Comment:
Did you have the chance to give this and the one right after this a try with the API?
B/c I think this filter is in the wrong place and should be erroring, since it filters on the user_has_role.expiry_date & user_has_permission.expiry_date, which do not exist.
You need to move them deeper in the nesting next to the `api_key` where you will be using `k` as the alias.